test/priority/priority.k

```diff
@@ -2,10 +2,19 @@ module PRIORITY
   imports INT
   imports BOOL
 
-  syntax S ::= "one" | "two" | "three" | "four" | "five" | "six" | "seven"
+  syntax S ::= "one" | "two" | "three" | "four" | "five" | "six" | "seven" | start(Int) | middle(Int)
   rule one => two [priority(1)]
   rule one => three [priority(2)]
 
+  rule start(X) => one       requires X <Int 0  [priority(1)]
+  rule start(X) => one       requires X >=Int 0 andBool X <Int 5  [priority(1)]
+  rule start(X) => middle(X) requires X <Int 10 [priority(2), label(first)]
```

Comment:
What happens to the anti-left of this rule when it is combined with the other rules?

Reply:
I'm not fully sure I understood what you meant. Please tell me if the paragraph below does not answer your question.

When `first` is combined with `second` or `third`, its antileft is unchanged. Both of the test output files above contain `/*  */ \not{SortGeneratedTopCell{}}(/*  */ priorityLE1{}())`, which should be exactly the antileft of `first`.